ui/packages/catalog-ui-search/src/main/webapp/component/select/filterHelper.js

```diff
@@ -14,30 +14,37 @@
  **/
 function matchesFilter(filterValue, str, matchcase) {
   filterValue = getAppropriateString(filterValue, matchcase)
+  filterValue = escapeRegExp(filterValue)
+  str = escapeRegExp(str)
   const reg = new RegExp('\\b' + filterValue + '.*')
```

Comment:
❓Will this still work now that `filterValue` is being escaped?

Reply:
Yes. The `filterValue` is the user's input since they will be filtering a drop down list. If they enter a `*` symbol as part of that input, we do not want that being processed as part of the regex itself.